ui/plugins/auto_classification/controller.js

```diff
@@ -264,8 +263,6 @@ treeherder.factory('ThStructuredLinePersist', ['$q',
 treeherder.factory('ThUnstructuredLinePersist', [
     'thExtendProperties', 'thNotify', 'ThTextLogSummaryLineModel',
     function(thExtendProperties, thNotify, ThTextLogSummaryLineModel) {
-        var ThUnstructuredLinePersist = function(thNotify, ThTextLogSummaryLineModel) {};
-
```

Comment:
I don't think you should actually remove this? I think doing so will break the autoclassify code.